ui/src/epinio/Credentials.js

```diff
@@ -13,9 +13,10 @@ function Credentials(props) {
   React.useEffect(() => {
     const getCredentials = async () => {
       try {
+	// note: `-l` returns a list, hence the `.items...`, even if only a single secret matches
         const result = await window.ddClient.extension.host.cli.exec(
           "kubectl",
-          ["get", "secret", "-n", "epinio", "default-epinio-user", "-o", "jsonpath={.data}"]
+          ["get", "secret", "-n", "epinio", "-l", "epinio.io/role=admin", "-o", "jsonpath={.items[*].data}"]
```

Comment:
I think this will return invalid JSON if there is more than one admin. Maybe better use 
```suggestion
          ["get", "secret", "-n", "epinio", "-l", "epinio.io/role=admin", "-o", "jsonpath={.items[0].data}"]
```

Reply:
Right. Agreed. Will fix.

Reply:
And done from the page.